agent/app/agent.go

```diff
@@ -1098,25 +1098,25 @@ func (agent *ecsAgent) startACSSession(
 }
 
 // validateRequiredVersion validates docker version.
-// Minimum docker version supported is 1.9.0, maps to api version 1.21
-// see https://docs.docker.com/develop/sdk/#api-version-matrix
+// Minimum docker version supported is 1.12.0, maps to api version 1.24
```

Comment:
nit (non-blocking): we should also update the comment on [L.320](https://github.com/aws/amazon-ecs-agent/blob/master/agent/app/agent.go#L320C2-L320C55) as well.